SampleUsages/TestSamples/Queue-CSharp-HighMemory/run.csx

```diff
@@ -4,9 +4,9 @@ public static void Run(string sizeMb, out string output, TraceWriter log)
 {
     var arrSize = int.Parse(sizeMb) * 1024 * 1024;
     var bytes = new byte[arrSize];
-    for (int i = 0; i < arrSize; i++)
+    for (int i = 0; i < arrSize; i += 256)
```

Comment:
You can iterate by page size 4096. Even lesser iterations and CPU time to help out here.